tst/EnergyPlus/unit/OutputProcessor.unit.cc

```diff
@@ -1062,31 +1062,31 @@ namespace OutputProcessor {
         EXPECT_TRUE(compare_eso_stream(delimited_string({"1,616771620.987027,4283136,12,21, 1,10,4283196,12,21,24,60"}, "\n")));
 
         WriteReportIntegerData(*state, 1, "1", 616771620.98702729, StoreType::Averaged, 10, ReportingFrequency::TimeStep, 0, 0, 0, 0);
-        EXPECT_TRUE(compare_eso_stream(delimited_string({"1,61677162.0987027"}, "\n")));
+        EXPECT_TRUE(compare_eso_stream(delimited_string({"1,61677162.098703"}, "\n")));
```

Comment:
Not complaining here but it seems {:.6R} is not the same as {:.6f} and using {:.7f} would result in no diffs?

Reply:
I'm not sure, but both of them feel super arbitrary, so I'm not going to investigate :smile: